src/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -1240,18 +1249,21 @@ public void VerifyHMAC_SHA256()
             sign.LoadXml(doc.DocumentElement["Signature"]);
 
             // verify MS-generated signature
-            Assert.True(sign.CheckSignature(new HMACSHA256(hmackey)));
+            Assert.True(sign.CheckSignature(new HMACSHA256(emptyHmacKey)));
```

Comment:
You could also add `Assert.False(sign.CheckSignature(new HMACSHA256(someOtherKey));`